sync-team/src/github/mod.rs

```diff
@@ -779,6 +882,92 @@ pub fn construct_branch_protection(
     }
 }
 
+/// Convert a branch protection to a ruleset with merge queue enabled
+pub fn construct_ruleset(
+    _expected_repo: &rust_team_data::v1::Repo,
+    branch_protection: &rust_team_data::v1::BranchProtection,
+) -> api::Ruleset {
+    use api::*;
+
+    let uses_merge_bot = !branch_protection.merge_bots.is_empty();
+    let branch_protection_mode = if uses_merge_bot {
+        BranchProtectionMode::PrNotRequired
+    } else {
+        branch_protection.mode.clone()
+    };
```

Comment:
no repository uses merge_bot as far as I can see.
This option is not even documented. Why using it?

Maybe I'm missing something.